client/lib/mixins/perf/why-did-you-update/index.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
If we were to commit this, we should credit the author: https://gist.github.com/joshma/6753333dd38a6010f9a6#file-whydidyouupdatemixin-js make sure its licence is GPLv2 compatible.